honeybee_radiance_command/_command.py

```diff
@@ -185,3 +185,95 @@ def after_run(self):
 
     def __repr__(self):
         return self.to_radiance()
+
+    @staticmethod
+    def path_checker(file_path=None, extn_list=[], file_descr=''):
```

Comment:
You should never use a mutable default value for an argument. See here: https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

Just use None and replace it with [] inside the code.

Reply:
Also in what case would file_path be None?

```suggestion
    def path_checker(file_path, extn_list=None, file_descr=None):
```

Reply:
@mostaphaRoudsari Actually the file path is None as the default argument for several methods that we have in classes. Examples: [1](https://github.com/ladybug-tools/honeybee-radiance-command/blob/master/honeybee_radiance_command/gendaymtx.py#L60-L65). But I agree that it need not be set as default None here.

Reply:
Hi @built-tools! I see why you need to handle the cases for None. Makes sense. I would still don't put None here as the default value.